integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiSample.java

```diff
@@ -381,17 +576,19 @@ public void tearDownAll() {
   private static void assertImageExistsAndPushIfNeeded() {
     String imageName = envMap.get("MODEL_IMAGE_NAME");
     String imageVer = "notset";
+    String decoration = (envMap.get("DO_CM") != null && envMap.get("DO_CM").equalsIgnoreCase("true"))  ? "CM-" : "";
+
     if (imageName.equals(MII_SAMPLE_WLS_IMAGE_NAME_V1)) {
-      imageVer = "WLS-v1"; 
+      imageVer = "WLS-" + decoration + "v1";
     }
-    if (imageName.equals(MII_SAMPLE_WLS_IMAGE_NAME_V2)) { 
-      imageVer = "WLS-v2"; 
+    if (imageName.equals(MII_SAMPLE_WLS_IMAGE_NAME_V2)) {
+      imageVer = "WLS-" + decoration + "v2";
     }
-    if (imageName.equals(MII_SAMPLE_JRF_IMAGE_NAME_V1)) { 
-      imageVer = "JRF-v1"; 
+    if (imageName.equals(MII_SAMPLE_JRF_IMAGE_NAME_V1)) {
+      imageVer = "JRF-" + decoration + "v1";
     }
-    if (imageName.equals(MII_SAMPLE_JRF_IMAGE_NAME_V2)) { 
-      imageVer = "JRF-v2"; 
+    if (imageName.equals(MII_SAMPLE_JRF_IMAGE_NAME_V2)) {
+      imageVer = "JRF-" + decoration + "v2";
     }
```

Comment:
This code could be mad a little more clear in newer versions of Java which accept strings in case statements - e.g. something like:
```
String decoration = (envMap.get("DO_CM") != null && envMap.get("DO_CM").equalsIgnoreCase("true"))  ? "CM-" : "";  
switch (imageName) {
  case MII_SAMPLE_WLS_IMAGE_NAME_V1 : 
    imageVer = "WLS-" + decoration + "-v1"; 
    break;
  ...
}
```